docs/examples/aws_vpc.rb

```diff
@@ -6,6 +6,11 @@
     cidr_block "10.0.1.0/24"
   end
 
+  aws_internet_gateway vpc_name do
+      action [:create, :attach]
```

Comment:
vpc_name should be something like "provisioning_ig" at L9 and "provisioning_vpc" at L12